openedx_caliper_tracking/kafka_utils.py

```diff
@@ -17,3 +17,4 @@ def get_kafka_producer_configurations():
 
     except AttributeError as ex:
         LOGGER.exception('Invalid or no configurations are provided for KafkaProducer: %s', str(ex))
+        raise
```

Comment:
Can we raise some specific exception here?

Reply:
`InvalidConfigurationsException` will be raised outside this function where this exception is caught.